server/services/votes/finalize_votes.service.js

```diff
@@ -0,0 +1,106 @@
+import errors from '@feathersjs/errors';
```

Comment:
Maybe rename to `votes-finalize` Or `companies/finalize-votes`? Not sure what the best API design here would be, not to mention that finalizing votes is

Reply:
I got the concept of finalizing votes from the figma designs. Maybe the term update votes is a better idea? We just need a way to submit Pre-votes that can't be changed and then move to final votes that shouldn't be changed after they're "finalized"